### PR TITLE
Page discipline

### PR DESCRIPTION
**Pull request by yazidsef**
i have added the responsive to this page

### CODE REVIEW

File: public/assets/css/discipline.css [hunk 1]
```diff
@@ -77,9 +84,165 @@
...
-    height: 80px;
+    height: 88px;
     letter-spacing: 2px;
     overflow: hidden;
     white-space: wrap;
     text-overflow: ellipsis;
+}
+
+
+/*   .formation-text p {
```
Comment (Benja-san) [inline public/assets/css/discipline.css:95]: remove dead code

File: public/assets/css/style.css [hunk 2]
```diff
@@ -62,27 +63,34 @@ body {
...
 2. Correct the inheritance of border color in Firefox. (https://bugzilla.mozilla.org/show_bug.cgi?id=190655)
 3. Ensure horizontal rules are visible by default.
 */
 
 hr {
-    height: 0; /* 1 */
-    color: inherit; /* 2 */
-    border-top-width: 1px; /* 3 */
+    height: 0;
+    /* 1 */
```
Comment (Benja-san) [inline public/assets/css/style.css:76]: dead code

File: public/assets/css/style.css [hunk 3]
```diff
@@ -62,27 +63,34 @@ body {
...
+    color: inherit;
+    /* 2 */
```
Comment (Benja-san) [inline public/assets/css/style.css:78]: dead code

File: public/assets/css/style.css [hunk 4]
```diff
@@ -62,27 +63,34 @@ body {
...
+    border-top-width: 1px;
+    /* 3 */
```
Comment (Benja-san) [inline public/assets/css/style.css:80]: dead code

**Review by Benja-san — CHANGES_REQUESTED** [review]

**Comment by yazidsef** [comment]
ok

**Pull request closed without merging**